src/components/Accordion/Accordion.tsx

```diff
@@ -48,9 +48,12 @@ const Accordion = ({ headingLevel = 'h2', isContained = false, children, classNa
 
   const panelMaxHeight = (index: number) => {
     if (expandedPanels.has(index)) {
-      return { maxHeight: panelRefs.current[index].current?.scrollHeight };
+      return {
+        maxHeight: panelRefs.current[index].current?.scrollHeight,
+        display: 'contents'
```

Comment:
Display contents can ahve some inconsistent browser support, can we test this in safari/chrome/firefox?